site/_posts/2021-02-16-orbitvu-fusionauth-story.md

```diff
@@ -0,0 +1,88 @@
+
+We are also starting to use FusionAuth internally. Orbitvu is not just a software development company but we also design and manufacture our automated photo studios. Because of that, we have a lot of different departments that utilize a number of internal systems like ERP, project management software, or Helpdesk. All of these can also benefit from FusionAuth integration.
+
+Regarding the user management, this part remains in our own application due to the fact we already had it done before we switched to FusionAuth. We're integrating with FusionAuth using its API and it works flawlessly.
+
+**Dan:** What benefits, if any, have you seen from having one view of your customer across multiple applications?
+
+**Maciej:** The obvious benefit of having an SSO solution is that our clients can seamlessly switch between our applications without the need to log in multiple times or even realizing they're now in a different system. Another part of the story is that we, as Orbitvu, can easily manage the user accounts in a central way. What FusionAuth brings here (when compared to our old solution) is the support for multiple protocols that make it possible to integrate more applications more easily and securely. Another thing I like very much with FusionAuth is the built-in logout functionality.
+
+> After the initial setup, there is not much to do with FusionAuth except upgrading from time to time, so this way it is way more cost-effective than the competing solutions that are sometimes extremely expensive when it comes to a bigger number of users.
+
+**Dan:** What internal systems have you hooked up to FusionAuth? Were there any challenges doing so?
+
+**Maciej:** We've integrated our helpdesk system (Freshdesk) and our manuals site (Screensteps). This was rather easy due to the standards provided by FusionAuth. We've also tested the integration with our ERP system which is an open-source solution Odoo. The proof-of-concept implementation we did work with FusionAuth using the OAuth2 protocol but before going further we need to resolve some other non-technical issues.
+
+**Dan:** What problems did we solve for you? And how were you solving them before FusionAuth?
+
+**Maciej:** Before Orbitvu has adopted FusionAuth we've used an SSO solution based on CAS (Central Authentication Service). It worked well for a long time and mostly for the traditional, monolithic web applications but in the era of SPA, React Native, microservices, and Kubernetes it became obvious that the technological debt regarding our Auth system is too big. This is where FusionAuth comes in. It has brought us solutions matching the newest trends like OAuth2 + OIDC or 2FA and has opened a whole bunch of new possibilities in terms of possible architectural solutions and technologies we were able to use. At the same time, it became possible to integrate our system with other providers thanks to the OAuth2 or SAML.
```

Comment:
in terms of ~possible~ architectural solutions and technologies we ~were~ **are** able to use.